### PR TITLE
Gpt improvement

### PR DESCRIPTION
**Pull request by kylexrich**
Changes
- https://github.com/kylexrich/coursegpt/issues/109
- https://github.com/kylexrich/coursegpt/issues/117
- https://github.com/kylexrich/coursegpt/issues/114
- https://github.com/kylexrich/coursegpt/issues/105
- https://github.com/kylexrich/coursegpt/issues/100

Tested on most questions in [here](https://docs.google.com/document/d/1_-ISip_2j6UQIamgJ3pLP5V6TQ_v7-dlLu-mvL8RtRg/edit), in addition to some assignment related ones, and everything was good!

### CODE REVIEW

File: client/src/components/molecules/ChatSection/ChatSection.js [hunk 1]
```diff
@@ -86,13 +112,13 @@ const ChatSection = ({ message }) => {
           <div className={styles.chatComponent} style={{ backgroundColor }}>
             <div className={styles.chatContent}>
               {message && ProfileIcon}
-              <div className={styles.msgContent}>
+              <div className={styles.textBlock}>
```
Comment (ajo01) [inline client/src/components/molecules/ChatSection/ChatSection.js:115]: can I ask why you changed this part?

**Review by ajo01 — APPROVED** [review]
cleaned up some of the code. looks good

File: client/src/components/molecules/ChatSection/ChatSection.js (hunk 1 above)
Comment (kylexrich) [reply client/src/components/molecules/ChatSection/ChatSection.js:115]: oh woops, didnt even realize there was already a class for it. These changes were for the newline configuration

**Comment by kylexrich** [comment]
> cleaned up some of the code. looks good

Thank you!

**Review by huangcar1 — APPROVED** [review]

**Review by ritikk7 — APPROVED** [review]

**Pull request merged**